wpilibj/build.gradle

```diff
@@ -54,13 +54,18 @@ compileJava {
     dependsOn generateJavaVersion
 }
 
+repositories {
+    jcenter()
+}
+
 dependencies {
     compile project(':hal')
     compile project(':wpiutil')
     compile project(':ntcore')
     compile project(':cscore')
     compile project(':cameraserver')
     testCompile 'com.google.guava:guava:19.0'
+    testCompile "org.mockito:mockito-core:2.+"
```

Comment:
This is a scary dependency.  Can you make the mock yourself?

Reply:
Its in test, why would that be a scary dependency? It wouldn't be required by the library.

Reply:
Actually, the way you are using it is ok.

It can be scary because if you use it to mock a dependency, it can provide a false sense of security.

Reply:
Mockito saves an absolute ton of work in making mocks.  I'm aware of the dangers in mocking objects that you don't own.